src/controllers/BaseController.ts

```diff
@@ -67,6 +67,10 @@ export abstract class BaseController {
 
         const browser = await Puppeteer.launch(options);
         const page = await browser.newPage();
+        // Set a sane user agent to avoid anti bot protections
```

Comment:
```suggestion
        // Set a same user agent to avoid anti bot protections
```

Reply:
I skipped this one. "Same" isn't really capturing intent. Maybe a full reword would be better? e.g. `Set a standard browser user agent ...`

Reply:
Hi! I'll be taking over this review from Ant. I think `Set a standard browser user agent to avoid anti bot protections` is good 👍